ts/Extensions/ExportData.ts

```diff
@@ -469,6 +471,71 @@ addEvent(Chart, 'render', function (): void {
     }
 });
 
+addEvent(Chart, 'afterViewData', function (): void {
+    const chart = this,
+        dataTableDiv = chart.dataTableDiv,
+        row = document.getElementsByClassName('highcharts-table-sort-row')[0],
+        getCellValue = function (
```

Comment:
Can use arrow function. That also applies to most or all of the functions below.